build_tools/check.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
```

Comment:
might be useful to add `set -x` here to make it more obvious which step failed.
We would definitely do this in CI (i.e. run `sh -x build_tools/check.sh`)